kong/plugins/statsd/log.lua

```diff
@@ -144,7 +144,11 @@ local get_workspace_id = {
 }
 
 local metrics = {
-  unique_users = function (scope_name, message, metric_config, logger, conf)
+  unique_users = function (scope_name, message, metric_config, logger, conf, tags)
+    if conf.tag then
```

Comment:
Could you highlight why we skip this in "tag mode"?

Reply:
and for `unique_users`, because of statsd add consumer information to other metrics tags, So I think it's not needed by tags mode.

Reply:
got it, please make sure to add this explanation as comments to the code